patches/cli/0002-Consolidate-Newtonsoft.Json-versions.patch

```diff
@@ -1,17 +1,17 @@
-From 860656146b1d7d882daf3ab6be3607604ab2d4e7 Mon Sep 17 00:00:00 2001
+From bd13401dcc8b22047ea57253620dc978c9a2bc44 Mon Sep 17 00:00:00 2001
 From: Chris Rummel <crummel@microsoft.com>
 Date: Mon, 11 Feb 2019 12:56:51 -0600
-Subject: [PATCH 4/7] Consolidate Newtonsoft.Json versions.
+Subject: [PATCH 2/6] Consolidate Newtonsoft.Json versions.
```

Comment:
What is this patch doing?  It doesn't look like it's changing anything.

Reply:
Removed this patch.